Makefile

```diff
@@ -1,4 +1,4 @@
-SOURCE ?= file go_bindata github aws_s3 google_cloud_storage
+SOURCE ?= file go_bindata github aws_s3 google_cloud_storage vfs
```

Comment:
Use the build contraint name `go_vfs`

Reply:
Done

Reply:
build constraint name should be `godoc_vfs`